spec/models/validators/date_not_beyond_first_of_next_month_validator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+include ActiveSupport::Testing::TimeHelpers
```

Comment:
I think it's better to add includes after the `RSpec.describe`. But in this case it makes no difference.